src/sagas/wallet.js

```diff
@@ -373,7 +373,7 @@ export function* loadTokens() {
   // spawn a new "thread" to handle it.
   //
   // `spawn` is similar to `fork`, but it creates a `detached` fork
-  yield spawn(fetchTokensMetadata, registeredTokens.map(token => token.uid));
+  yield spawn(fetchTokensMetadata, registeredTokens.map(token => token.uid).filter(token => token !== htrUid));
```

Comment:
Is it possible to have a unit test for this?

Reply:
I dont think the desktop wallet has unit tests yet, only e2e with cypress